be/src/exec/exec_node.cpp

```diff
@@ -848,4 +849,12 @@ Status ExecNode::exec_debug_action(TExecNodePhase::type phase) {
     return Status::OK();
 }
 
+void ExecNode::add_chunk_accumulate_operator_if_needed(OpFactories& ops, pipeline::PipelineBuilderContext* context,
+                                                       int id) {
+    // Only add chunk accumulate opearator for non stream pipeline context.
+    if (!context->is_stream_pipeline()) {
```

Comment:
I'm afraid finally we still need the `ChunkAccumulateOperator` for stream pipeline:
1. Data source could be fragmented, fire many small chunks
2. The epoch could be large, several seconds
3. Without accumulate data into a larger chunk, the interpretation and scheduling overhead would be significant

Such being the case, the `ChunkAccumulateOperator` should be aware of epoch:
1. Accumulate chunk within an epoch
2. Fire the accumulated chunk when epoch finished, like eos